src/System.IO.Ports/tests/SerialPort/GetPortNames.cs

```diff
@@ -46,7 +31,33 @@ private void OpenEveryPortName()
                 }
             }
         }
+
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/20588 - GetPortNames() has registry dependency.", TargetFrameworkMonikers.UapAot)]
+        private void AllHelperPortsAreInGetPortNames()
+        {
+            string[] serialPortNames = SerialPort.GetPortNames();
```

Comment:
I would add some comments that point to the regression that was caused to make it clear that this is a regression test.